docs/src/add-services/redis.md

```diff
@@ -31,61 +27,59 @@ Otherwise, they're identical.
+| {{< image-versions image="redis" status="supported" environment="grid" >}} | {{< image-versions image="redis" status="supported" environment="dedicated-gen-3" >}} | {{< image-versions image="redis" status="supported" environment="dedicated-gen-2" >}} |
 
 {{% image-versions-legacy "redis" %}}
 
 {{% deprecated-versions %}}
 
 | Grid | {{% names/dedicated-gen-3 %}} | {{% names/dedicated-gen-2 %}} |
 |------|-------------------------------|------------------------------ |
-|  {{< image-versions image="redis" status="deprecated" environment="grid" >}} | {{< image-versions image="redis" status="deprecated" environment="dedicated-gen-3" >}} | {{< image-versions image="redis" status="deprecated" environment="dedicated-gen-2" >}} |
+| {{< image-versions image="redis" status="deprecated" environment="grid" >}} | {{< image-versions image="redis" status="deprecated" environment="dedicated-gen-3" >}} | {{< image-versions image="redis" status="deprecated" environment="dedicated-gen-2" >}} |
 
-{{< note >}}
-Versions 3.0 and higher support up to 64 different databases per instance of the service, but Redis 2.8 is configured to support only a single database.
-{{< /note >}}
+Note that versions 3.0 and higher support up to 64 different databases per instance of the service,
+while Redis 2.8 only supports a single database.
 
 ## Service types
 
-There are two types of the Redis service depending on what you want to do with it.
+Depending on your needs,
+you can set up Redis as [ephemeral](#ephemeral-redis) or [persistent](#persistent-redis).
 
 ### Ephemeral Redis
 
-The ephemeral Redis service is configured to serve as a cache; its storage isn't persistent.
+By default, Redis is an ephemeral service that serves as a non-persistent cache.
+Ephemeral Redis stores data only in memory and requires no disk space.
+When the service reaches its memory limit, it triggers a cache cleanup.
+To customize those cache cleanups, set up an [eviction policy](#eviction-policy).
 
-Data in an ephemeral Redis instance is stored only in memory and thus requires no disk space.
-When the service hits its memory limit,
-it automatically evicts cache items according to the [configured eviction rule](#eviction-policy) to make room for new ones.
+When you use ephemeral Redis,
+make sure the cache is cleaned up regularly.
+To trigger a cache cleanup every time your app starts,
+configure a `start` [web command](../create-apps/app-reference.md#web-commands) similar to:
 
-Your app must not treat ephemeral Redis as permanent.
-Instead, the cache needs to be regenerated as necessary.
-For example, if a container is moved for a reason such as region maintenance,
-the `deploy` and `post_deploy` hooks don't run and an app that treats the cache as permanent shows errors.
```

Comment:
:lady_beetle: This information is probably necessary to include. The problem is that people don't view such events as things that would cause problems and then they run into issues and open support tickets. The new wording only suggests doing it regularly, which could be done by cron or something. A stronger push for the start command with reasons like these (though perhaps better phrased) would probably be good.